tools/plotting/bar_chart.py

```diff
@@ -43,7 +42,7 @@ def main(tmpFileName):
     try:
         in_file = open(sys.argv[1], 'r')
         xtic = int(sys.argv[2])
-        col_list = string.split(sys.argv[3], ",")
+        col_list = sys.argv[3].split(",")
```

Comment:
```suggestion
        col_list = sys.argv[3].split(',')
```
for the sake of consistency?